rcl_logging_spdlog/test/test_logging_interface.cpp

```diff
@@ -111,7 +117,7 @@ TEST_F(LoggingTest, init_old_flushing_behavior)
   RestoreEnvVar env_var("RCL_LOGGING_SPDLOG_EXPERIMENTAL_OLD_FLUSHING_BEHAVIOR");
   rcpputils::set_env_var("RCL_LOGGING_SPDLOG_EXPERIMENTAL_OLD_FLUSHING_BEHAVIOR", "1");
 
-  ASSERT_EQ(RCL_LOGGING_RET_OK, rcl_logging_external_initialize(nullptr, allocator));
+  ASSERT_EQ(RCL_LOGGING_RET_OK, rcl_logging_external_initialize(nullptr, nullptr, allocator));
```

Comment:
I think we should add more test cases with specifying the file name prefix argument, and check that file does exist in the log directory.